packages/client/src/components/SafeSettings.js

```diff
@@ -266,9 +265,14 @@ const AddSafeOwner = ({ web3, onCancel, onNext }) => {
 
   const onAddressChange = async (newAddress) => {
     setAddress(newAddress);
-    setAddressValid(isAddr(newAddress) && (await isAddressValid(newAddress)));
+    setAddressValid(isAddr(newAddress) && (await isAddressValid(newAddress)) && !isAddressExisting(safeOwners, newAddress));
   };
 
+  const isAddressExisting = (safeOwners, newAddress) => {
+    const address = newAddress.startsWith("0x") ? newAddress : `0x${newAddress}`;
```

Comment:
we can use this `formatAddress` defined here: https://github.com/DapperCollectives/VESSEL/blob/main/packages/client/src/utils.js#L45

Reply:
Nice catch, updated!